frontend/package.json

```diff
@@ -4,91 +4,91 @@
+    "react-dom": "^18.2.0",
+    "react-dropzone": "^14.2.3",
     "react-final-form": "^6.5.9",
-    "react-intl": "^6.2.5",
+    "react-intl": "^6.4.0",
     "react-meta-elements": "^1.0.0",
     "react-placeholder": "^4.1.0",
-    "react-redux": "^7.2.4",
+    "react-redux": "^8.0.5",
     "react-router-dom": "^6.10.0",
-    "react-scripts": "^4.0.3",
-    "react-select": "^5.7.0",
-    "react-tooltip": "^4.2.21",
+    "react-select": "^5.7.1",
+    "react-tooltip": "5.7.4",
     "reactjs-popup": "^2.0.5",
-    "redux": "^4.1.0",
+    "redux": "^4.2.1",
     "redux-persist": "^6.0.0",
-    "redux-thunk": "^2.3.0",
-    "sass": "^1.58.0",
+    "redux-thunk": "^2.4.2",
+    "sass": "^1.62.0",
     "short-number": "^1.0.7",
     "shpjs": "^4.0.4",
     "slug": "^8.2.2",
     "tachyons": "^4.12.0",
     "tributejs": "^5.1.3",
     "use-query-params": "^2.2.1",
     "webfontloader": "^1.6.28",
-    "workbox-core": "^6.1.5",
-    "workbox-expiration": "^6.1.5",
-    "workbox-precaching": "^6.1.5",
+    "workbox-core": "^6.5.4",
+    "workbox-expiration": "^6.5.4",
+    "workbox-precaching": "^6.5.4",
     "workbox-recipes": "^6.5.4",
-    "workbox-routing": "^6.1.5",
-    "workbox-strategies": "^6.1.5"
+    "workbox-routing": "^6.5.4",
+    "workbox-strategies": "^6.5.4"
   },
   "scripts": {
     "build-locales": "combine-messages -i './src/**/messages.js' -o './src/locales/en.json'",
     "copy-static": "bash -c \"mkdir -p public/static/id; mkdir -p public/static/rapid; if ! (test -a public/static/id/index.js); then cp -R node_modules/@hotosm/id/dist/* public/static/id; fi; if ! (test -a public/static/rapid/index.js); then if ! (test -a node_modules/RapiD/dist/RapiD.css) then mv node_modules/RapiD/dist/iD.css node_modules/RapiD/dist/RapiD.css; fi; cp -R node_modules/RapiD/dist/* public/static/rapid; fi\"",
     "update-static": "bash -c \"mkdir -p public/static/id; mkdir -p public/static/rapid; cp -R node_modules/@hotosm/id/dist/* public/static/id; if ! (test -a node_modules/RapiD/dist/RapiD.css) then mv node_modules/RapiD/dist/iD.css node_modules/RapiD/dist/RapiD.css; fi; cp -R node_modules/RapiD/dist/* public/static/rapid;\"",
     "preparation": "bash -c \"if (test -a ../tasking-manager.env); then grep -hs ^ ../tasking-manager.env .env.expand > .env; else cp .env.expand .env; fi\"",
     "start": "npm run preparation && npm run copy-static && react-scripts start",
-    "build": "npm run preparation && npm run update-static && react-scripts build",
+    "build": "npm run preparation && npm run update-static && react-scripts build --stats",
     "prettier": "prettier --write 'src/**/*.js'",
     "lint": "eslint src",
     "test": "npm run lint && react-scripts test --env=jsdom",
     "coverage": "npm run test -- --coverage --watchAll=false",
-    "analyze": "source-map-explorer 'build/static/js/*.js'"
+    "analyze": "source-map-explorer 'build/static/js/*.js' && webpack-bundle-analyzer build/bundle-stats.json"
```

Comment:
I did. I must have removed the `webpack-bundle-analyzer` when I was rebasing accidentally (it _was_ in `devDependencies`).

Reply:
To add on: I was using `webpack-bundle-analyzer` since `source-map-explorer` wasn't showing me everything (there was a blob in the main bundle that it didn't "explore").